palace/models/postoperator.cpp

```diff
@@ -985,8 +949,8 @@ auto PostOperator<solver_t>::MeasureAndPrintAll(int step, const ComplexVector &e
     int idx_pad = 1 + static_cast<int>(std::log10(num_conv));
     table.col_options = {6, 6};
     table.insert(Column("idx", "m", idx_pad, {}, {}, "") << step + 1);
-    table.insert(Column("f_re", "Re{f} (GHz)") << freq.real());
-    table.insert(Column("f_im", "Im{f} (GHz)") << freq.imag());
+    table.insert(Column("f_re", "Re{f} (GHz)") << omega.real());
+    table.insert(Column("f_im", "Im{f} (GHz)") << omega.imag());
```

Comment:
This table is just printed for logging so is in SI units. Dimensionalize with something maybe like this?

```suggestion
    table.insert(Column("f_re", "Re{f} (GHz)") <<  units.Dimensionalize<Units::ValueType::ENERGY>(omega.real()));
    table.insert(Column("f_im", "Im{f} (GHz)") <<  units.Dimensionalize<Units::ValueType::ENERGY>(omega.imag()));
```